lib/locale/nl.yml

```
@@ -0,0 +1,5 @@
+nl:
+  errors:
+    messages:
+      not_pwned: "eerder is verschenen in een datalek en mag niet worden gebruikt"
+      pwned_error: "kon niet worden geverifieerd aan de hand van eerdere datalekken"
```

```suggestion
      not_pwned: "is eerder verschenen in een datalek en mag niet worden gebruikt"
      pwned_error: "kon niet worden geverifieerd in eerdere datalekken"
```

I am nominally dutch (born and raised there but it's been 2 decades), I think this tweak is closer to how it should read.

---

Thanks & adjusted the commit & added you as co-author